db/schema.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE people
+(
+	ID 		INTEGER PRIMARY KEY AUTOINCREMENT,

```

Comment:
got some tab issues here.

Reply:
Well they don't look like that in my text editor. Maybe I just have to edit it on github?